### PR TITLE
Backend unit testing

### PR DESCRIPTION
**Pull request by dkancalade**
-- contains tests for database/index.js, database/dbHelper.js, and server/index.js
-- not all of the tests I wrote passed so I also cleaned up the functions that didn't pass so there are minor changes to a lot of files. 
-- did I successfully gitIgnore the coverage files?

### CODE REVIEW

**Review by mervinpan — COMMENTED** [review]
@dkancalade I think you should gitignore your coverage folder

File: database/seed.js [hunk 1]
```diff
@@ -54,14 +53,15 @@ db.once('open', function() {
...
     } else {
-      console.log('db insertion complete', response);
       db.close();
     }
   });
-});
\ No newline at end of file
+});
+
+module.exports.insertSeed;
```
Comment (mervinpan) [inline database/seed.js:67]: Not sure if this is being used.

File: spec/db.test.js [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+
+test('the function createRandomQuantity takes in a maximum number and does not return the same number when called multiple times', () => {
+  const max = 10;
+  const randomNumber1 = createRandomQuantity(max);
+  const randomNumber2 = createRandomQuantity(max);
+  const randomNumber3 = createRandomQuantity(max);
+  const randomNumber4 = createRandomQuantity(max);
+  const randomNumbers = [randomNumber2, randomNumber2, randomNumber3, randomNumber4];
+
+  expect(randomNumbers.every((val) => val === randomNumber1)).toBe(false);
```
Comment (mervinpan) [inline spec/db.test.js:15]: If your numbers are random, wouldn't this have a chance to be true?

**Review by mervinpan — APPROVED** [review]
Nothing critical. I just added some notes of things I noticed. Nice job getting Jest working!

File: spec/db.test.js (hunk 2 above)
Comment (dkancalade) [reply spec/db.test.js:15]: Yeah, that's true.  I figured it was low enough probability.  Maybe there is a recursive solution that can do a better job.  I'll make a ticket for tech debt to work on that when I have a chance.

File: database/seed.js (hunk 1 above)
Comment (dkancalade) [reply database/seed.js:67]: I'll check thanks.

**Pull request merged**